Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -51,32 +84,31 @@ def bump_formula_pr
       safe_system "brew", "update"
     end
 
-    Utils::Inreplace.inreplace(formula.path) do |s|
-      if new_url_hash
-        old_url = formula_spec.url
-        if ARGV.dry_run?
-          ohai "replace '#{old_url}' with '#{new_url}'"
-          ohai "replace '#{old_hash}' with '#{new_hash}'"
-        else
-          s.gsub!(old_url, new_url)
-          s.gsub!(old_hash, new_hash)
-        end
-      else
-        resource_specs = formula_spec.specs
-        old_tag = resource_specs[:tag]
-        old_revision = resource_specs[:revision]
-        if ARGV.dry_run?
-          ohai "replace '#{old_tag}' with '#{new_tag}'"
-          ohai "replace '#{old_revision}' with '#{new_revision}'"
-        else
-          s.gsub!(/['"]#{old_tag}['"]/, "\"#{new_tag}\"")
-          s.gsub!(old_revision, new_revision)
-        end
-      end
+    old_formula_version = formula_version(formula, requested_spec)
+
+    replacement_pairs = []
+    if new_url_hash
+      replacement_pairs << [formula_spec.url, new_url]
+      replacement_pairs << [old_hash, new_hash]
+    else
+      replacement_pairs << [formula_spec.specs[:tag], new_tag]
+      replacement_pairs << [formula_spec.specs[:revision], new_revision]
     end
```

Comment:
This could be simplified to (it gets a bit longer, but less repetition):

```ruby
replacement_pairs = if new_url_hash
  [
    [formula_spec.url, new_url],
    [old_hash, new_hash],
  ]
else
  [
    [formula_spec.specs[:tag], new_tag],
    [formula_spec.specs[:revision], new_revision],
  ]
end
```